configuration.go

```diff
@@ -46,10 +46,12 @@ const (
 	BundlePolicyMaxCompat
 )
 
-var BundlePolicyString = []string{"Balanced", "MaxBundle", "MaxCompat"}
-
 func (p BundlePolicy) String() string {
-	return EnumToStringSafe(int(p), BundlePolicyString)
+	return EnumToStringSafe(int(p), []string{
+		"Balanced",
```

Comment:
This looks cleaner, but I have a concern that passing inline allocates new memory for the full slice each time. Maybe this is a micro-optimization, but if we don't want to export the string slices, would just lower-casing the first character work?